apps/devhub/templates/devhub/api/agreement.html

```diff
@@ -0,0 +1,29 @@
+{% extends "devhub/addons/submit/base.html" %}
```

Comment:
it looks like the only thing different here between the [existing agreement page](https://github.com/Nolski/olympia/blob/signing-api-agreement/apps/devhub/templates/devhub/addons/submit/start.html) is the title, right? Can you maybe refactor them both to share a common base so there isn't any repeated content?